packages/odyssey-react-mui/package.json

```diff
@@ -50,7 +50,7 @@
     "build:source": "NODE_ENV=production BROWSERSLIST_ENV=modern babel src --out-dir dist --extensions .js,.jsx,.ts,.tsx --source-maps",
     "build:testSelectors": "tsx scripts/generateTestSelectorsJson.ts",
     "build:types": "tsc --project tsconfig.production.json",
-    "build": "yarn build:clean && yarn generate:i18n && yarn build:types && yarn build:source && yarn build:testSelectors",
+    "build": "yarn build:clean && yarn generate:i18n && yarn build:types && yarn build:source",
```

Comment:
Was this intentional or a mistake? Seems like could be a bad merge.

Reply:
Ah, this must have happened from the rebase, will re-add, thank you for catching this!